README.md

```diff
@@ -51,14 +51,24 @@ make plugins.json
 
 In addition to running as a standalone server, the Marketplace is also designed to run as a Lambda function, compiling the `plugins.json` database into the binary for immediate access without further configuration.
 
+### Automatic Deployment
+
+Changes merged to `master` are automatically deployed to https://api.staging.integrations.mattermost.com.
+
+Changes merged to `production` are automatically deployed to https://api.integrations.mattermost.com.
+
+To update production, simply create, review and merge the appropriate pull request. Often this will just be a merge from `master` to `production`, but if there are pending changes not yet ready for a production release there, it may be appropriate to just update the plugin database with a direct change to production that is later merged back to `master`.
```

Comment:
I would like to avoid having to merge `master` into `production` all to often. I was envisioning ~one merge per month.

Why should updates to `plugins.json` not always to to `production`?

Reply:
In my original proposal the wording was:
> Adding new releases to plugins or adding a new plugin should be submitted to `production`. **Any** other changes should go into `master`.

What I like about this is that it very clear, where every change should go.

Reply:
feel like once a month is slow, unless ofcourse we're pushing `plugins.json` more often directly to production then 👌

Reply:
Tweaked the proposal here -- thoughts?